src/reports/components/report-sections/report-collapsible-container.tsx

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { css } from '@uifabric/utilities';
+import { NamedFC } from 'common/react/named-fc';
+import * as React from 'react';
+
+import { CollapsibleComponentCardsProps } from '../../../DetailsView/components/cards/collapsible-component-cards';
+
+const ReportCollapsibleContainer = NamedFC<CollapsibleComponentCardsProps>('ReportCollapsibleContainer', props => {
+    const { id, header, headingLevel, content, containerClassName, buttonAriaLabel } = props;
+
+    const contentId = `content-container-${id}`;
+
+    const outerDivClassName = css('collapsible-container', containerClassName, 'collapsed');
+
+    const titleContainerProps = { role: 'heading', 'aria-level': headingLevel };
+
+    return (
+        <div className={outerDivClassName}>
+            <div className="title-container" {...titleContainerProps}>
```

Comment:
When I see `titleContainerProps` my expectation is a props for a `TitleContainer` React component, which confused me for a moment here. Also, I was distracted by the difference between unvarying `className="title-container` being specified inline but unvarying `role="heading"` being injected. Could it be simpler to write this as
```suggestion
            <div className="title-container" role="heading" aria-level={headingLevel}>
```
instead?

Reply:
Done